pkg/flow/states/action.go

```diff
@@ -46,9 +48,9 @@ func (logic *actionLogic) Deadline(ctx context.Context) time.Time {
 	d, err := duration.ParseISO8601(logic.Timeout)
 	if err != nil {
 		if logic.Timeout != "" {
-			logic.Log(ctx, "failed to parse timeout: %v", err)
+			logic.Log(ctx, util.Error, "failed to parse timeout: %v for %s", err, logic.label())
+			return time.Now().Add(DefaultLongDeadline)
```

Comment:
This return was shifted into a different location and I suspect that's incorrect?